src/Microsoft.DotNet.Darc/DarcLib/Helpers/ManifestHelper.cs

```diff
@@ -164,21 +173,23 @@ private static List<DownloadedAsset> SelectMergedManifestAssets(IEnumerable<Down
                 // it is taken from the Build.Name, which corresponds to the repository name.
                 assetOrigin ??= repoName;
 
-                if (assetOriginMap.TryGetValue(assetName, out var existingAssetOrigin))
+                if (assetMetadataMap.TryGetValue(assetName, out var existingAssetMetadata))
                 {
-                    if (existingAssetOrigin != assetOrigin)
```

Comment:
I'm not sure about this condition - in which case can we have the same asset coming from different repos @andriipatsula ? If I understand correctly now the 2nd occurence of the asset will have a `null` origin, but what if the second one is shipping. Then maybe we should pick the shipping one

Reply:
I was/am not sure if we will have just a single MergedManifest.xml file or multiple MergedManifests for each repo, plus a single one that combines all repo-level MergedManifests. As I am reading all "MergedManifests.xml" file in the drop `List<DownloadedAsset> mergedManifestAssets = SelectMergedManifestAssets(downloadedBuilds);`

We can remove this condition after we ensure that this situation is not possible.

Reply:
ok, I see. We can return to that after we have a published VMR build